sys-block/targetcli-fb/targetcli-fb-2.1.53.ebuild

```diff
@@ -3,7 +3,7 @@
 
 EAPI=7
 
-PYTHON_COMPAT=( python3_{7,8} )
+PYTHON_COMPAT=( python3_{7,8,9} )
```

Comment:
Change with PYTHON_COMPAT=( python3_{8,9} ) as python3.7 is not used anymore. Also try 10 if it works